wp-parsely.php

```diff
@@ -1046,15 +1071,65 @@ public function insert_parsely_javascript() {
 		/**
 		 * Filters whether to include the Parsely JavaScript file.
 		 *
-		 * If true, the file is included.
+		 * If true, the JavaScript files are sourced.
 		 *
 		 * @since 2.2.0
 		 *
 		 * @param bool $display True if the JavaScript file should be included. False if not.
 		 */
-		if ( apply_filters( 'parsely_filter_insert_javascript', $display ) ) {
-			include 'parsely-javascript.php';
+		if ( ! apply_filters( 'parsely_filter_insert_javascript', $display ) ) {
+			return;
+		}
+
+		wp_register_script(
+			'wp-parsely-api',
+			plugin_dir_url( __FILE__ ) . 'build/init-api.js',
+			[ 'wp-polyfill-fetch' ],
+			self::get_asset_cache_buster(),
+			true
+		);
+
+		add_filter( 'script_loader_tag', [ $this, 'script_loader_tag' ], 10, 3 );
+
+		$dependencies = array();
+
+		if ( ! empty( $parsely_options['api_secret'] ) ) {
+			$dependencies[] = 'wp-parsely-api';
+			wp_localize_script(
+				'wp-parsely-api',
+				'wpParsely', // This globally-scoped object will hold our initialization variables
+				array(
+					'apikey' => $parsely_options['apikey'],
+				)
+			);
+		}
+
+		wp_enqueue_script(
+			'wp-parsely-tracker',
+			'https://cdn.parsely.com/keys/' . esc_url( $parsely_options['apikey'] ) . '/p.js',
```

Comment:
Escaping of a URL is handled by WP when it outputs the HTML (late), so it's not needed here.